src/openPr.js

```diff
@@ -93,6 +95,25 @@ module.exports = async function ({ context, inputs, packageVersion }) {
       },
       inputs
     )
+
+    // attach artifact
+    const buildDir = inputs['release-artifact-build-folder']
+
+    if (buildDir) {
+      logInfo('** Attaching artifact **')
+
+      const token = inputs['github-token']
+      const { id: releaseId } = draftRelease
+      try {
+        await attachArtifact(buildDir, releaseId, token)
+      } catch (err) {
+        logInfo(err.message)
+        core.setFailed(err.message)
+      }
+
+      logInfo('** Artifact attached! **')
+    }
```

Comment:
I wonder if it wouldn't be more appropriate to do this in the app instead of the action. I'm not sure, just checking